apps/src/gamelab/GameLabJr.interpreted.js

```diff
@@ -184,12 +184,27 @@ function makeNewSprite(animation, x, y) {
     } else if (position === "top center") {
       sprite.x = 200;
       sprite.y = 50;
-    } else if (position === "bottom right") {
+    } else if (position === "top right") {
       sprite.x = 350;
-      sprite.y = 350;
+      sprite.y = 50;
+    } else if (position === "center left") {
+      sprite.x = 50;
+      sprite.y = 200;
+    } else if (position === "center") {
+      sprite.x = 200;
+      sprite.y = 200;
     } else if (position === "center right") {
       sprite.x = 350;
       sprite.y = 200;
+    } else if (position === "bottom left") {
+      sprite.x = 50;
+      sprite.y = 350;
+    } else if (position === "bottom center") {
+      sprite.x = 200;
+      sprite.y = 350;
+    } else if (position === "bottom right") {
+      sprite.x = 350;
+      sprite.y = 350;
```

Comment:
I wonder if it's cleaner to have the x and y location as the dropdown value, so the library doesn't need any knowledge of "top left" etc.